scalalib/overrides-2.11/scala/concurrent/impl/AbstractPromise.scala

```diff
@@ -0,0 +1,29 @@
+package scala.concurrent.impl
+
+/**
+ * JavaScript specific implementation of AbstractPromise
+ *
+ * This basically implements a "CAS" in Scala for JavaScript. Its
+ * implementation is trivial because there is no multi-threading.
+ *
+ * @author Tobias Schlatter
+ */
+abstract class AbstractPromise {
+
+  private var state: AnyRef = _
+
+  protected final
+  def updateState(oldState: AnyRef, newState: AnyRef): Boolean = {
+    if (state eq oldState) {
+      state = newState
+      true
+    } else false
+  }
+
+  protected final def getState: AnyRef = state
+
+}
+
+object AbstractPromise {
+  protected def updater = ???
+}
```

Comment:
Is our override logic not able to cope with this file existing in `overrides` and `overrides-2.12`?